app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityPresenter.kt

```diff
@@ -221,4 +281,12 @@ class QuestionPlayerActivityPresenter @Inject constructor(
       TAG_HINTS_AND_SOLUTION_DIALOG
     ) as? HintsAndSolutionDialogFragment
   }
+
+  /** Set reading text size normal. */
```

Comment:
```suggestion
  /** Set reading text size to normal, which is the default. */
```

This is more clear, as I rember having clarity issues in one of my initial review passes.